packages/cli-e2e/utils/cli.ts

```diff
@@ -67,4 +74,41 @@ export function setupUIProject(
   return buildProcess;
 }
 
+export function getConfig() {
+  const pathToConfig = resolve(
+    __dirname,
+    ...new Array(4).fill('..'),
+    '.config',
+    '@coveo',
+    'cli',
+    'config.json'
+  );
```

Comment:
What is the path you're trying to reach here?
(please put it in absolute :P) 

I have the feeling this is what causing issue in the E2E.